LanguageEngine/Summarizer/Gensim.py

```diff
@@ -0,0 +1,93 @@
+import warnings
+warnings.filterwarnings(action='ignore', category=UserWarning, module='gensim')
+
+from gensim.summarization.summarizer import summarize
+from gensim.summarization import keywords
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+import re
+
+def removeRefNo(inputString):
+    """Removes reference numbers from the string
```

Comment:
mention that it is the wikipedia reference numbers just to make it clear
```suggestion
    """Removes reference numbers from the strings obtained from wikipedia
```